main/idf_component.yml

```diff
@@ -1,10 +1,16 @@
 dependencies:
   # Required IDF version
   idf: ">=5.1"
-
-  mdns: "^1.1.0"
-  chmorgan/esp-libhelix-mp3: "1.0.3"
-  esp-dsp: "^1.3.4"
+  espressif/esp32-camera: "^2.0.5"  
```

Comment:
can you use the latest git of esp32-camera instead?

Reply:
yes, I can set the PACMAN to get it from github master.